src/System.Management.Automation/utils/ClrFacade.cs

```diff
@@ -209,7 +210,7 @@ internal static Encoding GetOEMEncoding()
             if (s_oemEncoding == null)
             {
 #if UNIX        // PowerShell Core on Unix
-                s_oemEncoding = GetDefaultEncoding();
+                s_oemEncoding = PowerShellEncoding.GetDefaultEncoding();
 #elif CORECLR   // PowerShell Core on Windows
```

Comment:
I think the CORECLR branches could use commenting in both GetDefaultEncoding and GetOEMEncoding. The logic reasoning for GetACP and GetOEMCP is opaque.